tests/baselines/reference/mappedTypeRecursiveInference.errors.txt

```diff
@@ -1,7 +1,4 @@
+tests/cases/compiler/mappedTypeRecursiveInference.ts(19,14): error TS2589: Type instantiation is excessively deep and possibly infinite.

```

Comment:
I don't know how to handle this case properly, see the discussion [here](https://github.com/microsoft/TypeScript/pull/52737#discussion_r1168757127)